MAUI/ImageEditor/getting-started.md

```diff
@@ -307,4 +307,98 @@ The image editor utilizes the `AspectFit` image scaling of Image control to fit
 {% endhighlight %}
 {% endtabs %}
 
-N> The size value will only be available after the image has been loaded into view.
\ No newline at end of file
+N> The size value will only be available after the image has been loaded into view.
+
+## ImageEditor inside stack layout
+
+### VerticalStackLayout
+
+The image editor is placed inside vertical stack layout, and users can specify a minimum height request, with the default value already predefined as 100.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<VerticalStackLayout>
+    <imageEditor:SfImageEditor x:Name="imageEditor"
+                               HeightRequest="400"
+                               Source="image.jpeg">
+    </imageEditor:SfImageEditor>
+</VerticalStackLayout>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+public MainPage()
+{
+    InitializeComponent();
+    VerticalStackLayout verticalLayout = new VerticalStackLayout();
+    SfImageEditor imageEditor = new SfImageEditor();
+    imageEditor.Source = ImageSource.FromResource("MyProject.Resources.Images.image.jpeg");
+    imageEditor.HeightRequest = 400;
+    verticalLayout.Add(imageEditor);
+    this.Content = verticalLayout;
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### HorizontalStackLayout
+
+The image editor is placed inside horizontal stack layout, and users can specify a minimum width request, with the default value already predefined as 100.

```

Comment:
When the image editor is placed inside a horizontal stack layout, users must define the required `MinimumWidthRequest` value. By default, this `MinimumWidthRequest`  is set to 100.

Reply:
Updated